src/EditorFeatures/CSharpTest/UsePatternCombinators/CSharpUsePatternCombinatorsDiagnosticAnalyzerTests.cs

```diff
@@ -88,6 +94,7 @@ public async Task TestMissingOnExpression(string expression)
             await TestAllMissingOnExpressionAsync(expression);
         }
 
+        [InlineData("i == default || i > default(int)", "i is default(int) or > (default(int))")]
```

Comment:
simplifier doesn't handle relational patterns. I didn't go through that - probably best to be done separately. /cc @CyrusNajmabadi